src/Component/tests/Reflection/CallableReflectionTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Resource\Tests\Reflection;
+
+use PHPUnit\Framework\TestCase;
+use ReflectionFunctionAbstract;
+use Sylius\Component\Resource\Tests\Dummy\RepositoryWithCallables;
+use Sylius\Resource\Reflection\CallableReflection;
+
+final class CallableReflectionTest extends TestCase
+{
+    public function testItIsInitializable(): void
+    {
+        $callableReflection = new CallableReflection();
+        $this->assertInstanceOf(CallableReflection::class, $callableReflection);
+    }
+
+    public function testItReflectsAnArrayCallable(): void
+    {
+        $reflection = CallableReflection::from([RepositoryWithCallables::class, 'find']);
+        $this->assertInstanceOf(ReflectionFunctionAbstract::class, $reflection);
```

Comment:
```suggestion
        $this->assertInstanceOf(\ReflectionFunctionAbstract::class, $reflection);
```
Also, its import should be removed

Reply:
done